config/locales/views/admin/en.users.yml

```diff
@@ -12,6 +12,8 @@ en:
         subhead: You may use this to edit external users. The email address and username must match.
         update: Update
         internal: Internal Pricing
+      show:
+        unexpire: Un-expire
```

Comment:
TIL that there’s no verb unexpire, and that unexpired is just an adjective (I went looking after this sounded a bit off to my ears). Maybe consider `remove expiration` as the action here, since it seems like that’s what we’re doing?

If you prefer unexpire as a verb, that’s cool too, but Apple’s Dictionary says there’s no dash in it 😄

Reply:
I think that's probably why I put the dash in there because it's not really a word. And possibly influenced by "re-activated". 

I like "remove expiration".